docs/contributor/ui.md

```diff
@@ -0,0 +1,53 @@
+# BTP Manager User Interface
+
+> [!WARNING]
+> This feature is in the experimental stage and is not yet available in the main branch or official releases.
+> Use the latest development image to test the UI: europe-docker.pkg.dev/kyma-project/dev/btp-manager:PR-720
+
+Follow steps below to run BTP Manager with UI:
+1. Connect `kubectl` to your cluster by setting `KUBECONFIG` environment variable.
+```shell
+export KUBECONFIG=<path-to-kubeconfig>
+```
```

Comment:
Please indent the code blocks and other content under each of the steps.

Reply:
Suggestion applied.